presto-ml/src/test/java/io/prestosql/plugin/ml/TestLearnAggregations.java

```diff
@@ -51,18 +50,14 @@
 
 public class TestLearnAggregations
 {
-    private static final TypeManager typeManager;
+    private static final Metadata METADATA = createTestMetadataManager();
+    private static final TypeManager typeManager = METADATA.getTypeManager();
 
     static {
-        TypeRegistry typeRegistry = new TypeRegistry();
+        TypeRegistry typeRegistry = (TypeRegistry) typeManager;
```

Comment:
Is there a better way to do this?

Reply:
No.  This is the only test that needs to manually register a type.